lib/util/jsonata.ts

```diff
@@ -0,0 +1,18 @@
+import jsonata from 'jsonata';
+import * as memCache from './cache/memory';
+
+export function getExpression(input: string): jsonata.Expression | Error {
```

Comment:
I think you need tests here in utils, otherwise CI will fail with coverage issues